op-e2e/actions/interop/interop_txplan_test.go

```diff
@@ -291,6 +291,108 @@ func reorgOutUnsafeAndConsolidateToSafe(t helpers.Testing, actors *dsl.InteropAc
 	require.Equal(t, unsafeHeadNumAfterReorg+1, replacedBlock.Number)
 }
 
+// reorgOutUnsafeAndConsolidateToSafeBothChain assumes both chainX and chainY are reorged.
+// chain{X|Y} both expected to experience cross-unsafe invalidation and reorging unsafe blocks.
+// Consolidate with steps: unsafe -> cross-unsafe -> local-safe -> safe
+func reorgOutUnsafeAndConsolidateToSafeBothChain(t helpers.Testing, actors *dsl.InteropActors, chainX, chainY *dsl.Chain, startX, startY, endX, endY, unsafeHeadNumAfterReorg uint64) {
+	require.GreaterOrEqual(t, endY, unsafeHeadNumAfterReorg)
+	// Check to make batcher happy
+	require.Positive(t, endY-startY)
+	require.Positive(t, endX-startX)
```

Comment:
This will underflow as `uint64` and always be positive. You may instead want to use `require.Less(startX, endX)`, and same for Y.